### PR TITLE
Change the default command from printing s2i instructions to starting bash.

### PR DESCRIPTION
**Pull request by tmds**
Fixes https://github.com/redhat-developer/s2i-dotnetcore/issues/390.

### CODE REVIEW

File: 6.0/runtime/default-cmd.sh [hunk 1]
```diff
@@ -1,9 +1,3 @@
 #!/bin/bash
 
-cat <<EOF
-This is a runtime image for .NET ${DOTNET_CORE_VERSION}.
-
-See the documentation at https://github.com/redhat-developer/s2i-dotnetcore/tree/master/${DOTNET_CORE_VERSION}/runtime.
-
-This image can also be used as an S2I image for pre-compiled applications. Run 's2i usage <image>' for more information.
-EOF
+exec /bin/bash "$@"
```
Comment (tmds) [inline 6.0/runtime/default-cmd.sh:3]: fyi, the CMD in the runtime image starts this script.

The script is there so s2i assemble can replace it, and the resulting s2i image will directly start the .NET app.

**Review by omajid — APPROVED** [review]
LGTM!

**Review by aslicerh — APPROVED** [review]
:shipit:

**Comment by tmds** [comment]
Thanks for the quick reviews!

**Pull request merged**